snapcraft/tests/commands/test_collaborate.py

```diff
@@ -84,9 +106,23 @@ def test_collaborate_bad_request(self):
         self.client.login('dummy', 'test correct password')
         err = self.assertRaises(
             storeapi.errors.StoreValidationError,
-            collaborate,
+            _store.collaborate,
             'badrequest', 'keyname')
 
         self.assertEqual(
                 'Received error 400: "The given `snap-id` does not match '
                 'the assertion\'s."', str(err))
+
+    def test_collaborate_unchanged_collaborators(self):
+        self.edit_collaborators_mock.return_value = [{
+            'developer-id': 'test-dev-id',
+            'since': '2017-02-10T08:35:00.390258Z',
+            'until': '2018-02-10T08:35:00.390258Z'
+        }]
```

Comment:
There's a bug here. _edit_collaborators will always return the times without milliseconds. So we will never hit this case in real life.
The comparison needs to ignore milliseconds.